cetmix_tower_server/tests/test_variable_option.py

```diff
@@ -0,0 +1,76 @@
+from odoo.tests import TransactionCase
+
+
+class TestTowerVariableOption(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.variable = self.env["cx.tower.variable"].create(
+            {
+                "name": "odoo_versions",
+            }
+        )
+
+    def test_create_variable_option(self):
+        """Test creation of a TowerVariableOption record."""
+        option = self.env["cx.tower.variable.option"].create(
+            {
+                "name": "14.0",
+                "variable_id": self.variable.id,
+            }
+        )
+        self.assertEqual(option.name, "14.0")
+        self.assertEqual(option.variable_id, self.variable)
+
+    def test_create_with_default_context(self):
+        """Test creation of a TowerVariableOption using
+        default_variable_id in context."""
+        with self.env.cr.savepoint():
```

Comment:
Why do we need a savepoint here?

Reply:
To ensure that changes are made in that test,but i will delete that test